File: src/main/java/dev/shota/decompiler/window/menu/edit/Edit.java
package dev.shota.decompiler.window.menu.edit;

import dev.shota.decompiler.window.utils.Language;
import javax.swing.*;
import java.awt.event.KeyEvent;

public class Edit extends JMenu {

    public Edit() {
        setText(Language.get("edit"));
        setMnemonic(KeyEvent.VK_E);

        add(new EditCopy());
        add(new EditSelectAll());
    }

}

File: src/main/kotlin/dev/shota/decompiler/window/menu/edit/Edit.kt
package dev.shota.decompiler.window.menu.edit

import dev.shota.decompiler.window.menu.Menu
import dev.shota.decompiler.window.menu.edit.items.Copy
import dev.shota.decompiler.window.menu.edit.items.Find
import dev.shota.decompiler.window.menu.edit.items.SelectAll
import java.awt.event.KeyEvent

class Edit : Menu("edit", KeyEvent.VK_E) {

    init {
        add(Copy())
        add(SelectAll())
        add(Find())
    }

}